pages/ordeal-bubble-chart.vue

```diff
@@ -0,0 +1,48 @@
+<template>
```

Comment:
I'd reported in the original Ordeal PR that the cookie banner needed to be changed from absolute positioning as it broke on pages that were long and could be scrolled. Is it possible to re-add the fix into this branch please? Cheers!

Reply:
done

Reply:
Thanks Ruby. Confirmed as working